### PR TITLE
Floating Edge Check Enhancements

### PR DESCRIPTION
**Pull request by savannahostrowski**
This pull request makes changes solely to the instruction message associated with Floating Edge check flags.

### CODE REVIEW

**Comment by savannahostrowski** [comment]
@medercc If you have anything else to document for other enhancements, please write them here so I can keep track! 😄

**Comment by medercc** [comment]
Confirmed that removing the single quotations has fixed the OSM ID formatting.

**Comment by savannahostrowski** [comment]
@medercc Any other enhancements to make to the Floating Edge check? Or are we ready to close out development work and pass it up for review?

**Comment by medercc** [comment]
@savannahostrowski No other enhancements needed at this time.

**Comment by savannahostrowski** [comment]
@danielduhh I feel silly making you review this but we are ready for review ;)

File: src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/FloatingEdgeCheck.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
     public static final double DISTANCE_MINIMUM_METERS_DEFAULT = 100;
     // create a simple instruction stating the Edge with the supplied OSM Identifier is floating.
     private static final List<String> FALLBACK_INSTRUCTIONS = Arrays
-            .asList("Way '{0,number,#}' is floating. Ie. has no incoming or outgoing ways.");
+            .asList("Way {0,number,#} is floating. Ie. has no incoming or outgoing ways.");
```
Comment (danielduhh) [inline src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/FloatingEdgeCheck.java]: i.e.?

Comment (savannahostrowski) [reply src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/FloatingEdgeCheck.java]: Oops, fixed it.

**Pull request closed without merging**